src/python/geoclaw/surge/storm.py

```diff
@@ -78,17 +82,16 @@ def __init__(self, path=None, file_format="hurdat2", **kwargs):
         self.max_wind_radius = None
         self.central_pressure = None
         self.storm_radius = None
+        self.new_year = None  
```

Comment:
Via our conversation earlier I think we do not need to keep track of this if we just default to the beginning of the track.